### DISCUSSION TITLE
Can you use this package in an existing Laravel app?

### DISCUSSION THREAD

**Discussion by jamesblackwell** [Kyon147/laravel-shopify#236] [Q&A] [answered]
I have a functioning web app that I would like to embed in Shopify. Is it possible with this app? 

Specifically, I still want to maintain the ability for non-Shopify users to sign up and use the app standalone.

**Comment by JonPurvis** [comment, marked as answer]
Hey @jamesblackwell 

I know this is over 2 years old but I just wanted to let you know that it is possible. One of my apps has a page that can be accessed via the web, as a normal Laravel application and then the Shopify Admin looks at a specific part of my application.
